src/Collection/Iterator/SortIterator.php

```diff
@@ -65,12 +65,12 @@ public function __construct($items, $callback, $dir = SORT_DESC, $type = SORT_NU
 
         $callback = $this->_propertyExtractor($callback);
         $results = [];
-        foreach ($items as $key => $value) {
-            $value = $callback($value);
-            if ($value instanceof DateTimeInterface && $type === SORT_NUMERIC) {
-                $value = $value->format('U');
+        foreach ($items as $key => $val) {
+            $val = $callback($val);
+            if ($val instanceof DateTimeInterface && $type === SORT_NUMERIC) {
+                $val = $value->format('U');
```

Comment:
You missed renaming one instance of $value here.

Reply:
Whoops.